src/components/TodoInput.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+
+export default class TodoInput extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: this.props.todoText,
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.addItem = this.addItem.bind(this);
+        this.formSubmiting = this.formSubmiting.bind(this);
```

Comment:
որպեսզի այսպես ամեն մի մեթոդի համար `this`֊ը `bind` չանես , կարող ես օգտագործել `arrow function`

Reply:
Ամեն անգամ իրեն սարքելուց խուսափելու համար arrow function չեմ գրել